Makefile.am

```diff
@@ -98,6 +98,17 @@ AM_CPPFLAGS = \
 	-DPACKAGE_SYSCONF_DIR=\""$(sysconfdir)"\" \
 	$(NULL)
 
+AM_CFLAGS = \
+	-Wall \
+	-Werror=strict-prototypes \
```

Comment:
Note to self: Apparently all of them are valid for clang as well, the unit test is happy.

Reply:
This patch is actually broken: it's overridden via the various `*_CFLAGS` variables that we define per-target.  I've dropped this for now, and will bring it back as part of a more comprehensive cleanup that I've been preparing.